[SC-357] Dodanie obsługi markdownów w wytycznych

dmncp opened the pull request:
W wytycznych mamy pole `infoContent`, w którym daje się dłuższy opis. Fajnie by było jakby on obsługiwał markdowny (w json editorze oraz w widoku wytycznych).
PS. w json editorze ofc nie można dodawać stringów multiline, więc żeby dodawać nowe linie trzeba robić `/n` zamiast enterów. Jest to z jednej strony oczywiste, ale może być mylące bo w mardownie nową linię robimy poprzez `<br />`

dmncp committed:
[SC-357] feature: markdown text support for info task

kwdrt commented:
Wydaje mi się że nie działa prawidłowo:
![firefox_Iaxp0Qrtzk](https://user-images.githubusercontent.com/56168780/201490155-ab2d6bc5-ff72-489b-8ba5-3b3b243d1fc4.png)


Przydałby się przykład markdowna w opisie, bo nie udało mi się przekazać działającego, może to ja czegoś nie widzę

dmncp commented:
@kwdrt to nie wina markdowna że ci nie przeszło tylko jest bug w edycji info taska. Zobacz tutaj punkt 4:
https://docs.google.com/document/d/1LkdHjQXUgemxjFhUQRRd46U_lMuhxlasLr52hqrR3lg/edit?usp=sharing

Moja wina bo miałem o tym napisać ale zapomniałem. Jak spojrzysz w treść błędu dokładniej to widać że dotyczy obrazków. No i występuje przed tymi zmianami z tego pr

dmncp commented:
A na screenie podałeś poprawnego markdowna ;) Jak sprawdzisz nie w edycji a w dodawaniu to zadziała

kwdrt commented:
Ok, to daję approve w takim razie - jeśli dodamy automatycznie to będzie git

dmncp committed:
[SC-357] Merge remote-tracking branch 'origin/develop' into SC-357-dodanie-obslugi-markdownow-w-wytycznych

# Conflicts:
#	frontend/src/components/student/AllActivities/InfoTask/Information.js

dmncp committed:
[SC-357] refactor: infoContent card height fixed
